src/commands/configureLaunchDarklyEnvironment.ts

```diff
@@ -1,36 +1,31 @@
 import { commands, Disposable, window } from 'vscode';
-//import { sortNameCaseInsensitive } from '../api';
-import { extensionReload } from '../utils';
-import { LDExtensionConfiguration } from '../ldExtensionConfiguration';
+import { extensionReload } from '../generalUtils';
+import { CMD_LD_CONFIG_ENV } from '../utils/commands';
+import { ILDExtensionConfiguration } from '../models';
 
-export default async function configureEnvironmentCmd(config: LDExtensionConfiguration): Promise<Disposable> {
-	const configureEnvironmentCmd = commands.registerCommand(
-		'launchdarkly.configureLaunchDarklyEnvironment',
-		async () => {
-			try {
-				//const intConfig = config.getConfig();
-				const api = config.getApi();
-				if (!config.getConfig()) {
-					return;
-				}
-				const project = await api?.getProject(config.getConfig().project);
-				if (!project) {
-					window.showErrorMessage(`[LaunchDarkly] Please Configure LaunchDarkly Extension`);
-					return;
-				}
-				//const environments = project.environments.sort(sortNameCaseInsensitive);
-				const environments = project.environments.items;
-				const newEnvironment = await window.showQuickPick(environments.map((env) => env.key));
-				if (newEnvironment) {
-					await config.getConfig().update('env', newEnvironment, false);
-					await extensionReload(config, true);
-				}
-			} catch (err) {
-				console.log(err);
-				window.showErrorMessage(`[LaunchDarkly] ${err}`);
+export default async function configureEnvironmentCmd(config: ILDExtensionConfiguration): Promise<Disposable> {
+	const configureEnvironmentCmd = commands.registerCommand(CMD_LD_CONFIG_ENV, async () => {
+		try {
+			const api = config.getApi();
+			if (!config.getConfig()) {
+				return;
 			}
-		},
-	);
+			const project = await api?.getProject(config.getConfig().project);
+			if (!project) {
+				window.showErrorMessage(`[LaunchDarkly] Please Configure LaunchDarkly Extension`);
```

Comment:
You might be able to do this in a few places but I'm not too picky.
```suggestion
				window.showErrorMessage(`${CONST_LD_PREFIX} Please Configure LaunchDarkly Extension`);
```